pkg/adaptor/cloud/libvirt/libvirt.go

```diff
@@ -78,7 +67,7 @@ func createCloudInitISO(v *vmConfig, libvirtClient *libvirtClient) string {
 	return cloudInitIso
 }
 
-func checkInstanceExistsByName(name string, libvirtClient *libvirtClient) (exist bool, err error) {
+func checkDomainExistsByName(name string, libvirtClient *libvirtClient) (exist bool, err error) {
```

Comment:
Nit: s/checkDomainExistsByName/CheckDomainExistsByName  - "C" capital letter

Reply:
It's private function, hence not in capital

Reply:
No worries :-)